docs/_custom-rule-tutorial-code/foo-bar.js

```diff
@@ -0,0 +1,43 @@
+"use strict";
+
+// The foo-bar rule definition
+module.exports = {
+    meta: {
+        type: "fix",
+        docs: {
+            description: "Can only assign 'bar' to `const foo`.",
```

Comment:
```suggestion
            description: "Enforce that a variable named `foo` can only be assigned a value of 'bar'.",
```

Reply:
Note to self: apply to copied example in the docs as well